docs/sources/operators-guide/configure/configuring-out-of-order-samples-ingestion.md

```diff
@@ -64,3 +52,19 @@ Similar to the problem above with query caching, the samples recorded via the re
 So you should expect some difference in results if you happen to run the raw query of the recording rule. The difference highly depends on your out-of-order ingestion pattern.
 
 If you happen to have a shorter `out_of_order_time_window`, say less than 10 minutes, then you can use `-ruler.evaluation-delay-duration` to delay your rule evaluation up to that time.
+
+## Meaning of out-of-order
+
+Grafana Mimir and the Prometheus TSDB understand out-of-order as follows.
+
+The moment that a new series sample arrives, Mimir need to determine if the series already exists, and whether or not the sample is too old:
+
+- If the series exists, the incoming sample must have a newer timestamp than the latest sample that is stored for the series.
+  Otherwise, it is considered out-of-order and will be dropped by the ingesters.
+- If the series does not exist, then the sample has to be within bounds, which go back 1 hour from TSDB's head-block max time (when using 2 hour block range). If it fails to be within bounds, then it is also considered out-of-bounds and will be dropped by the ingesters.
```

Comment:
This has nothing to do with out-of-order.  One bit of code ends up solving two distinct problems for a user - out of order, or out of bounds.

Reply:
Do you suggest I remove this point?

Reply:
I think I can extend this point mentioning this solution solves this problem as well.